clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/EnvIntegrationSpec.scala

```diff
@@ -37,8 +55,41 @@ abstract class EnvIntegrationSpec(env: String)
    * to talk through the OpenIDC proxy.
    */
   override lazy val bearerToken: String = {
-    val vaultUtil = new VaultUtil(Env.Test)
-    val credential = vaultUtil.credentialForScopes(authScopes)
+    val vaultToken: String = vaultTokenFiles
+      .find(_.exists)
+      .map { file =>
+        new String(Files.readAllBytes(file.toPath)).stripLineEnd
+      }
+      .getOrElse {
+        sys.error("Vault token not found on filesystem!")
+      }
+
+    val vaultConfig = new VaultConfig()
+      .address(vaultUrl)
+      .token(vaultToken)
+      .build()
+
+    val vaultDriver = new Vault(vaultConfig)
+    val accountJSON =
+      vaultDriver
+        .logical()
+        .read(vaultPath)
+        .getData
+        .asScala
+        .toMap[String, String]
+        .asJson
+
+    val serviceAccount: ServiceAccount =
+      accountJSON
+        .as[ServiceAccount]
+        .fold({ err =>
+          throw new RuntimeException(
+            s"Failed to decode service account JSON from Vault at $vaultPath",
+            err
+          )
+        }, json => json)
```

Comment:
can you replace `X => X` with `_`?

Reply:
_ doesn't work, but I found a pre-defined `identity` that I'll try out.